dialogue-annotations-processor/src/main/java/com/palantir/dialogue/annotations/processor/data/ParamTypesResolver.java

```diff
@@ -49,7 +50,8 @@ public final class ParamTypesResolver {
             Request.PathParam.class,
             Request.QueryParam.class,
             Request.QueryMap.class,
-            Request.Header.class);
+            Request.Header.class,
+            HeaderMap.class);
```

Comment:
please match the above using `Request.HeaderMap.class`

Reply:
changed